tools/build.py

```diff
@@ -644,7 +646,7 @@ def adb_shell(*args, **kwargs):
 
 
 def _run_ios_tests(args, config: str, config_build_dir: Path):
-    if platform.machine() == "x86_64" and args.apple_arch != "x86_64":
+    if platform.machine() == "x86_64" and args.apple_arch != "x86_64" and args.apple_sysroot != "macosx":
```

Comment:
this function only gets called when `args.ios` is true - does it make sense for `args.apple_sysroot` to ever be "macosx" here?

Reply:
thanks for the catch up. fixed.